.github/workflows/deploy-to-prod.yml

```diff
@@ -12,6 +12,8 @@ env:
   KUBE_TOKEN: ${{ secrets.PROD_KUBE_TOKEN }}
 
   IMAGE_TAG: ${{ github.ref_name }}
+  ECR_REGISTRY: ${{ vars.ECR_REGISTRY }}
+  ECR_REPOSITORY: ${{ vars.PRODUCTION_ECR_REPOSITORY }}
```

Comment:
Might be showing my lack of Github Action knowledge here, but where do the values for the `vars` context come from? 👀

Reply:
I only recently learnt there was a variables tab in the GitHub Actions UI > https://github.com/ministryofjustice/operations-engineering-reports/settings/variables/actions